docs/fundamentals/jcloud/faq.md

```diff
@@ -27,3 +27,11 @@
   ```
 
   If you don't see any obvious errors, please raise an issue in [JCloud](https://github.com/jina-ai/jcloud/issues/new/choose)
+
+- **How do I configure JCloud log level when it's integrated in my system?**
```

Comment:
Let's not add a new section for this. We can mention below command in the above section.

```bash
export JCLOUD_LOGLEVEL=DEBUG && jc deploy flow.yml
```

Reply:
ok